packages/odyssey-react/src/components/Tooltip/Tooltip.theme.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  TipBorderRadius: theme.BorderRadiusBase,
+  TipPaddingBlock: theme.SpaceRemXs,
+  TipPaddingInline: theme.SpaceRemS,
+  TipTextColor: theme.ColorTextBodyInverse,
+  TipBackgroundColor: theme.ColorTextBody,
+  TipFontSize: theme.FontSizeCaption,
+  TipFontWeight: theme.FontWeightBold,
+  TipLineHeight: theme.FontLineHeightTitle,
+  TipBorderWidth: theme.BorderWidthBase,
+
+  TailWidth: theme.SpaceRemXs,
+  TailMargin: theme.SpaceRemS,
+
+  TransitionDuration: theme.TransitionDurationBase,
+  TransitionTiming: theme.TransitionTimingBase,
+  TransitionDelay: "1s",
```

Comment:
These look really good nice work, their application above in the style module is great too